test.lua

```diff
@@ -781,6 +781,62 @@ do
     end
 end
 
+---- REPL tests ----
+print("Running tests for REPL completion...")
+local wrapRepl = function()
+    local replComplete
+    local replSend = coroutine.wrap(function()
+        local output = {}
+        fennel.repl({
+            readChunk = function()
+                local chunk = coroutine.yield(output)
+                output = {}
+                return chunk and chunk .. '\n' or nil
+            end,
+            onValues = function(xs)
+                table.insert(output, xs)
+            end,
+            registerCompleter = function(completer)
+                replComplete = function(input)
```

Comment:
Shouldn't this just be `replComplete = completer`? Or are we specifically trying to get rid of multiple values?

Reply:
...yup. Poor form on my part, I'd added that when serializing when so I could sort first, since I'd noticed the order was wrong when serializing, but forgot to remove the wrapper when I changed it back. D'oh